include/Zydis/Internal/SharedData.h

```diff
@@ -879,8 +879,8 @@ typedef struct ZydisInstructionDefinitionMVEX_
 typedef struct ZydisAccessedFlags_
 {
     ZydisCPUFlagAction action[ZYDIS_CPUFLAG_MAX_VALUE + 1];
-    ZyanU32 flags_read      ZYAN_BITFIELD(22);
```

Comment:
Given that these are bitfields anyways, can't we just split out the FPU flags here as well, then get rid of the manual masks in `Decoder.c` entirely, leaving it to the compiler to deal with?

Reply:
Yes, already planned. Requires some additional generator changes so I did it quick&dirty for now.

Will improve this before we merge the PR.